terraform/modules/self-service/iam.tf

```diff
@@ -209,3 +212,7 @@ resource "aws_iam_role_policy_attachment" "task_access_metadata_bucket_attachmen
   role       = "${aws_iam_role.self_service_task.name}"
   policy_arn = "${aws_iam_policy.access_config_metadata.arn}"
 }
+
+data "aws_ssm_parameter" "integration_metatdata_bucket" {
```

Comment:
And again (metatdata).  Is this deliberate?

Reply:
nope that's totally a typo! thanks :)